packages/ui/src/primitives/tailwind/Toggle/index.tsx

```diff
@@ -29,22 +29,41 @@ import Label from '../Label'
 
 export interface ToggleProps {
   value: boolean
+  size?: 'sm' | 'md' | 'lg'
   label?: string
+  labelClassName?: string
+  containerClassName?: string
   className?: string
   onChange: (value: boolean) => void
   disabled?: boolean
 }
 
-const Toggle = ({ className, label, value, onChange, disabled }: ToggleProps) => {
+const sizeMap = {
+  md: 'w-11 h-6 after:w-5 after:h-5 after:top-[2px] after:start-[2px]',
+  lg: 'w-16 h-9 after:w-7 after:h-7 after:top-[4px] after:start-[5px]'
+}
+
+const Toggle = ({
+  containerClassName,
+  className,
+  labelClassName,
+  size,
+  label,
+  value,
+  onChange,
+  disabled
+}: ToggleProps) => {
   const twClassName = twMerge(
-    "peer relative h-6 w-11 cursor-pointer rounded-full bg-gray-200 after:absolute after:start-[2px] after:top-[2px] after:h-5 after:w-5 after:rounded-full after:border after:border-gray-300 after:bg-white after:transition-all after:content-['']",
+    "peer relative cursor-pointer rounded-full bg-gray-200 after:absolute after:rounded-full after:border after:border-gray-300 after:bg-white after:transition-all after:content-['']",
     'peer-checked:bg-blue-600 peer-checked:after:translate-x-full peer-checked:after:border-white peer-focus:ring-4 peer-focus:ring-blue-300 dark:border-gray-600 dark:bg-gray-700 dark:peer-focus:ring-blue-800',
     'peer-disabled:pointer-events-none peer-disabled:opacity-50',
-    className
+    className,
+    sizeMap[size ?? 'md']
   )
+  const containerTwClassName = twMerge('flex items-center justify-center gap-4', containerClassName)
```

Comment:
`justify-center` should not be added

![image](https://github.com/EtherealEngine/etherealengine/assets/55396651/f18c7263-2dc7-4583-9787-e76aa36ab897)

because these toggles are getting centered instead of being on the left